astroquery/esa/hubble/core.py

```diff
@@ -250,13 +260,83 @@ def get_artifact(self, artifact_id, *, filename=None, verbose=False):
         None. It downloads the artifact indicated
         """
 
-        params = {"RETRIEVAL_TYPE": "PRODUCT", "ARTIFACTID": artifact_id, "TAPCLIENT": "ASTROQUERY"}
+        return self.download_file(file=artifact_id, filename=filename, verbose=verbose)
+
+    def get_associated_files(self, observation_id, *, verbose=False):
```

Comment:
Not the same, in this case we are only listing the file names, in case users want to filter them before download anything. get_obs_products downloads them.